config/environments/development.rb

```diff
@@ -73,4 +73,6 @@
 
   # Allow connections from inside a docker container to this host machine
   config.hosts += %w[host.docker.internal]
+
+  config.force_ssl = false
```

Comment:
I think this shouldn't be necessary. The default when running in development environment is false, no need to declare this.
So if it is only working by declaring it, there is something else going on. If you take a look at the datastore and apply `development.rb` we didn't have to add this 🤔

Reply:
Yes I noticed but it's because it is set [in application.rb ](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/blob/main/config/application.rb#L38) so it needs to be overridden